playwright/e2e/clusters/cluster-archives-list.spec.ts

```diff
@@ -0,0 +1,86 @@
+      await clusterListPage.clickClusterListTableHeader('Status');
+      await clusterListPage.clickClusterListTableHeader('Provider (Location)');
+      await clusterListPage.scrollClusterListPageTo('bottom');
+      await clusterListPage.itemPerPage().click();
+      await clusterListPage.clickPerPageItem('20');
+      await clusterListPage.itemPerPage().click();
+      await clusterListPage.clickPerPageItem('50');
+      await clusterListPage.itemPerPage().click();
+      await clusterListPage.clickPerPageItem('100');
+      await clusterListPage.itemPerPage().click();
+      await clusterListPage.clickPerPageItem('10');
+    });
+
+    test('Cluster archives page : navigations', async ({ clusterListPage }) => {
+      await clusterListPage.showActiveClusters().click();
+      await clusterListPage.isClusterListScreen();
+      await clusterListPage.viewClusterArchives().click();
+    });
+
+    test('Cluster archives page : filter options', async ({ clusterListPage }) => {
+      await expect(clusterListPage.filterTxtField()).toBeVisible();
+      await clusterListPage.filterTxtField().click();
+      await clusterListPage.clickClusterTypeFilters();
+      await clusterListPage.clickClusterTypes('OCP');
+      await clusterListPage.filterTxtField().click();
+      await clusterListPage.clusterListRefresh();
+      await clusterListPage.waitForArchiveDataReady();
+
+      // Only OCP cluster should be in filter rule display
+      await clusterListPage.checkFilteredClusterTypes('OCP', true);
+      await clusterListPage.checkFilteredClusterTypes('OSD', false);
+      await clusterListPage.checkFilteredClusterTypes('ROSA', false);
+
+      // Only OCP clusters should be in cluster list - first page
+      await clusterListPage.checkFilteredClustersFromClusterList('OCP', true);
+      await clusterListPage.scrollClusterListPageTo('bottom');
+      await clusterListPage.goToLastPage();
+
+      // Only OCP clusters should be in cluster list - last page
+      await clusterListPage.checkFilteredClustersFromClusterList('OCP', true);
+      await clusterListPage.scrollClusterListPageTo('top');
+      await clusterListPage.waitForArchiveDataReady();
+      await clusterListPage.clickClusterTypeFilters();
+      await clusterListPage.clickClusterTypes('OSD');
+      await clusterListPage.clickClusterTypes('ROSA');
+      await clusterListPage.clickClusterTypes('ARO');
+      await clusterListPage.clickClusterTypes('OCP');
+      await clusterListPage.clearFilters();
+
+      await clusterListPage.filterTxtField().scrollIntoViewIfNeeded();
+      await expect(clusterListPage.filterTxtField()).toBeVisible();
+      await clusterListPage.filterTxtField().click();
+      await clusterListPage.filterTxtField().clear();
+      await clusterListPage.filterTxtField().fill('smoke cluster');
+      await clusterListPage.filterTxtField().clear();
+      await clusterListPage.waitForArchiveDataReady();
```

Comment:
i might be missing something, but where are all the assertions?

it seems that some tests just do a bunch of sequential steps (which `await` on each other - which is kinda red-flag on its own), but don't actually check the resulted state 🤔 ❓

Reply:
In this E2E scenario, the test performs a sequence of user interactions such as selecting and deselecting cluster types, entering filter values, and clearing inputs. Any issues during these interactions will naturally surface as test failures, so explicit assertions after every step are not always necessary.
That said, this flow is based on an existing Cypress definition, and the scope of this PR is to convert the same behavior to Playwright. If we want to introduce scope changes or expand test coverage, that should be addressed separately in a different PR.